### DISCUSSION TITLE
Hello, I have some suggestions regarding SEO.。你好，针对 SEO 有一些建议

### DISCUSSION THREAD

**Discussion by nishima-hikigaya** [pengzhanbo/vuepress-theme-plume#152] [Ideas]
建议1：请问是否有计划在page增加面包屑导航？
Suggestion 1: May I ask if there are plans to add breadcrumb navigation to the page?

建议2：分类和标签是否支持增加描述，样式可以参考：https://ahrefs.com/blog/category/keyword-research/ 或者 https://www.semrush.com/blog/category/marketing/ecommerce/
Suggestion 2: Do categories and tags support adding descriptions? For style references, you can look at: https://ahrefs.com/blog/category/keyword-research/ or https://www.semrush.com/blog/category/marketing/ecommerce/

祝好
Best regards

**Comment by pengzhanbo** [comment]
1. 面包屑导航 已添加在 `rc.91` 版本中。
2. 分类和标签页 描述，我暂时没有合适的 UI 方案去给 描述的内容腾出合适的空间布局，但我在这两个页面新增了 几个插槽，对于有此类需求的，可以通过 [布局插槽](https://theme-plume.vuejs.press/guide/layout-slots/#layout-%E6%8F%92%E6%A7%BD) 的方式 自行增加 描述功能。
